internal/sdkprovider/service/kafkaschema/kafka_schema.go

```diff
@@ -20,8 +20,9 @@ import (
 	"github.com/aiven/terraform-provider-aiven/internal/schemautil/userconfig"
 )
 
-// newlineRegExp is a regular expression that matches a newline.
-var newlineRegExp = regexp.MustCompile(`\r?\n`)
+// whitespaceRegExp is a regular expression to match any whitespace, new line, or carriage return
+// characters in a string. This is used to normalize Protobuf strings to compare them for logical equivalence.
+var whitespaceRegExp = regexp.MustCompile(`\s`)
```

Comment:
It's `\s+` to match more than one.

Reply:
Please see my reply to the above comment: https://github.com/aiven/terraform-provider-aiven/pull/1625#discussion_r1504169514